src/less/button/base/button.less

```diff
@@ -79,7 +79,7 @@ a.fake-btn--primary {
 }
 
 /* show hover states only for non-disabled or non-loading state button */
-button.btn--primary:not([disabled]):not([aria-live="polite"][aria-disabled="true"]),
```

Comment:
I saw this recently in code and thought it might be an issue. Apparently, it was.

Reply:
Unfortunately I don't think this change is quite right. It has broken the hover/focus state for loading buttons. 

```
// button[aria-live="polite"] is a button with a potential loading/busy state
// button[aria-live="polite"][aria-disabled="true"] is a button in a loading/busy state
```

Try changing the loading button example on the website from `btn` to `btn btn--primary` to see what I mean.

Reply:
Also, whatever change we go with, I think it will need applying to all button priorities, not just `primary`...